networking/network_policy/logging-network-policy.adoc

```diff
@@ -0,0 +1,44 @@
+[id="logging-network-policy"]
+= Logging network policy events
+include::modules/common-attributes.adoc[]
+:context: logging-network-policy
+
+toc::[]
+
+As a cluster administrator, you can configure network policy audit logging for your cluster and enable logging for one or more namespaces.
+
+[NOTE]
+====
+Audit logging of network policies is available for only the xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes cluster network provider].
+====
+
+include::modules/nw-networkpolicy-concept.adoc[leveloffset=+1]
+
+== Network policy audit configuration
+
+The configuration for audit logging is specified as part of the OVN-Kubernetes cluster network provider configuration. The following YAML illustrates default values for network policy audit logging feature.
+
+.Audit logging configuration
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  defaultNetwork:
+    ovnKubernetesConfig:
+      policyAuditConfig:
+        destination: "null"
+        maxFileSize: 50
+        rateLimit: 20
+        syslogFacility: local0
+----
+
+The following table describes the configuration fields for network policy audit logging.
+
+include::modules/nw-operator-cr.adoc[tag=policy-audit]
+
+include::modules/nw-networkpolicy-audit-configure.adoc[leveloffset=+1]
+include::modules/nw-networkpolicy-audit-enable.adoc[leveloffset=+1]
+include::modules/nw-networkpolicy-audit-disable.adoc[leveloffset=+1]
```

Comment:
I have a personal pref to add a blank line between includes, but it's clearly not a req.